package.json

```diff
@@ -69,12 +73,19 @@
     "tomchentw-npm-dev": "^3.1.0"
   },
   "peerDependencies": {
-    "fbjs": "^0.3.1",
+    "fbjs": "^0.6.0",
     "react": "^0.14.0"
   },
   "dependencies": {
     "classnames": "^2.1.5",
     "react-addons-update": "^0.14.0",
     "react-dom": "^0.14.0"
+  },
+  "babel": {
```

Comment:
I've had issues with .babelrc when importing into projects that use Babel v5. But I put it back into `.babelrc`.